src/thunderbots/software/util/parameter/parameter.h

```diff
@@ -233,6 +251,10 @@ class Parameter
     // Store the namespace of the parameter
     std::string namespace_;
 
+    std::mutex callback_mutex_;
```

Comment:
could we group with with the `value_mutex_`?

Reply:
fixed